tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "commonjs",
+    "resolveJsonModule": true,
+    "outDir": "./dist",
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    // Allow JS must be included to ensure that the built binary is included
+    // in the output. This could be copied directly in the future if required.
+    "allowJs": true,
+    "strict": false,
+    // TODO: 
+    // In order to gracefully move our project to TypeScript without having 
+    // TS immediately yell at you, we'll disable strict mode for now.
+    "strict": false,
+    "skipLibCheck": true,
```

Comment:
Is it actually safe to disable the lib check?

Reply:
https://www.typescriptlang.org/tsconfig#skipLibCheck

We can remove this and it will default to true

Reply:
Ok, if it's the default then I guess it's fine.